mobius-android/src/test/java/com/spotify/mobius/android/ObservableMutableLiveDataTest.java

```diff
@@ -0,0 +1,97 @@
+/*
+ * -\-\-
+ * Mobius
+ * --
+ * Copyright (c) 2017-2020 Spotify AB
+ * --
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * -/-/-
+ */
+package com.spotify.mobius.android;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.equalTo;
+
+import androidx.arch.core.executor.testing.InstantTaskExecutorRule;
+import androidx.lifecycle.Lifecycle;
+import com.spotify.mobius.android.MobiusLoopViewModelTestUtilClasses.TestModel;
+import com.spotify.mobius.disposables.Disposable;
+import java.util.ArrayList;
+import java.util.List;
+import org.junit.Rule;
+import org.junit.Test;
+
+public class ObservableMutableLiveDataTest {
+  @Rule public InstantTaskExecutorRule rule = new InstantTaskExecutorRule();
+
+  private final FakeLifecycleOwner lifecycleOwner1 = new FakeLifecycleOwner();
```

Comment:
Kind of out of the scope for this PR, maybe we could replace this with `LifecycleRegistry` in all tests?

Reply:
Can have a look at that, may make separate PR.

Reply:
Actually I'm not sure that the lifecycle registry can directly substitute the fake lifecycle owner, that is required in observe methods. The Fake Lifecycle Owner is more or less a wrapper around the `LifecycleRegistry`, but necessarily implements the `LifecycleOwner` interface that's required by `observe` calls. 

I do remember now, when I was originally writing this that this is more or less the standard way for testing, not sure if you had something else in mind.